src/index.ts

```diff
@@ -175,6 +173,7 @@ class Monika extends Command {
 
   async run() {
     const { flags } = this.parse(Monika)
+    getPublicIp() // calling it here to give time getting public IP before calling on alert functions
```

Comment:
i think this should wait until resolve. also it should be after the createConfig, because createConfig does not need the ip address.

Reply:
I have tried using async await, but the down side is the test will run longer because there are many test that running monika. What do you think?

Reply:
How about mock getIP when testing? Always return 192.168.0.1?

Reply:
Nice suggestion. Will try it

Reply:
If you want to have mock IP, it would be better to use class A IP (ex. 127.0.0.1) rather than class C.

Reply:
Fixed using mock Ip by adding env check in the function, because i'm stuck adding mock in the test file 😆